src/declarations/stencil-public-compiler.ts

```diff
@@ -2186,6 +2188,11 @@ export interface OutputTargetBase {
   type: string;
 }
 
+// TODO(NOW): rename
+export interface ValidatableOutputTarget extends OutputTargetBaseNext {
```

Comment:
Maybe `OutputTargetValidationConfig`/`OutputTargetValidationOptions`? Or if we wanted to represent it as the "base options" for output target validation: `OutputTargetBaseValidationConfig` (although I dislike the length of this type at that point).